services/kommander-appmanagement/0.2.0/defaults/cm.yaml

```diff
@@ -11,6 +11,6 @@ data:
         manager:
           replicas: "${kommanderAppManagementReplicas}"
           image:
-            tag: "${appManagementImageTag}"
+            tag: "${appManagementImageTag:=v2.2.0-dev-amd64}"
```

Comment:
As discussed, we need to include the outcomes of https://mesosphere.slack.com/archives/C02Q9AP62NN/p1646213450807939 here as we need to introduce new variables. Old ones can't be changed in 2.2.